engine/src/test/java/io/zeebe/engine/processor/workflow/subprocess/EventSubprocessTest.java

```diff
@@ -195,13 +223,39 @@ private void assertEventSubprocessLifecycle(long worklowInstanceKey) {
             tuple(WorkflowInstanceIntent.ELEMENT_COMPLETED, "event_sub_proc"));
   }
 
-  private static BpmnModelInstance createEventSubProc() {
+  private long deployAndWaitForJob(BpmnModelInstance model) {
```

Comment:
IMO, the relevant part is that the method triggers the event-subprocess.

```suggestion
  private long createInstanceAndTriggerEvent(BpmnModelInstance model) {
```